build.cmd

```diff
@@ -3,6 +3,12 @@ cd %~dp0
 
 SETLOCAL
 SET CACHED_NUGET=%LocalAppData%\NuGet\NuGet.exe
+IF NOT DEFINED BUILDCMD_KOREBUILD_VERSION (
+    SET BUILDCMD_KOREBUILD_VERSION=""
+)
+IF NOT DEFINED BUILDCMD_DNX_VERSION (
+    SET BUILDCMD_DNX_VERSION=""
```

Comment:
line 7 & 10 would be replaced with the exact versions by the pinning tool when we branch out to release/master